gstools/field/base.py

```diff
@@ -147,16 +147,15 @@ def mesh(
                 mesh.point_data[name] = field
         return out
 
-    def vtk_export(
-        self, filename, field_select="field", fieldname="field"
+    def _to_vtk_helper(
+        self, filename=None, field_select="field", fieldname="field"
     ):  # pragma: no cover
-        """Export the stored field to vtk.
+        """Create a VTK/PyVista grid of the stored field.
+
+        This is an internal helper that will handle saving or creating objects
 
         Parameters
         ----------
-        filename : :class:`str`
-            Filename of the file to be saved, including the path. Note that an
-            ending (.vtr or .vtu) will be added to the name.
```

Comment:
I guess you accidentally deleted the parameter description of "filename". Please undo the deletion and add a description for the different handling of None / not None.